src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drowable {
```

Comment:
is there anything common between figures which can be stored here, in abstract class? like `name` and `color`?